en/exercises/threat_identification/threat_analysis_survey.md

```diff
@@ -0,0 +1,73 @@
+||||||
+|**PRIVACY THREATS**|||||
+|1. Telephone eavesdropping (via telephone bugs, inductive sensors, or service providers|||||
+|2. Electromagnetic eavesdropping |||||
+|3. Rubbish eavesdropping (analyzing waste for confidential documents, etc.)|||||
+|4. Planted bugs in the building |||||
+||||||
+|**INTEGRITY/ACCURACY THREATS**|||||
+|1. Deliberate damage of information by external source|||||
+|2. Deliberate damage of information by internal sources|||||
+|3. Deliberate modification of information|||||
+||||||
+|**ACCESS CONTROL THREATS**|||||
+|1. Password cracking (access to password files, use of default/weak passwords, etc)|||||
+|2. External access to password files, and sniffing of the networks|||||
+|3. Unsecured maintenance of online services, developer backdoors|||||
+|4. Bugs in network software which can open unknown/unexpected security holes (holes can be exploited from externally to gain access)|||||
+|5. Unauthorized physical access to system|||||
+||||||
+|**LEGAL THREATS**|||||
+|1. Failure to comply with legal requirements|||||
+|2. Liability for acts of internal users or attackers who abuse the system to perpetrate unlawful acts (ie, incitement to racism, gambling, money laundering, distribution of pornographic or violent material)|||||
+|3. Liability for damages if an internal user attacks other sites|||||
+||||||
+|**RELIABILITY OF SERVICE THREATS**|||||
+|1. Major natural disasters, fire, water, earthquake, floods, power outages, etc|||||
+|2. Minor natural disasters, of short duration, or causing little damage|||||
+|3. Equipment failure from defective hardware, cabling, or communications system.|||||
+|4 Denial of Service due to network abuse: Misuse of routing protocols to confuse and mislead systems|||||
+|5. Downloading of malicious Applets, Active X controls, macros, PostScript files, etc through the browsers|||||
+|6. Sabotage: Physical destruction of network interface devices, cables|||||
+
+
+
+
+**Risk = Impact * Likelihood**
+
+##### SCALE
+|Impact Scale|Likelihood|
+|--------------------------------------------|------------------------------------------|
+|Impact is negligible =1|Unlikely to occur =0||||
+|Effect is minor, major organization operations are not affected=2| Likely to occur less than once per year =1||||
+|Organization operations are unavailable for a certain amount of time, costs are incurred. Public/customer confidence is minimally affected =3| Likely to occur once per year =2||||
+|Significant loss of operations, significant impact on public/customer confidence =4| likely to occur once per month =3||||
+|Effect is disastrous, systems are down for an extended period of time, rebuilding and replacement of systems is required =5| Likely to occur once per week =4||||
+|Effect is catastrophic, critical systems are completely down for an extended period; data is lost or irreparably corrupted; public and customers are totally affected =6| Likely to occur daily =5||||
+
+
```

Comment:
Same here: We can't use copy right content https://books.google.com/books?id=D5w3BwAAQBAJ&pg=PA200&lpg=PA200&dq=Ignorance:+inadequate+security+awareness,+lack+of+security+guidelines,+lack+of+proper+documentation,+lack+of+knowledge&source=bl&ots=VO6hjtp0qV&sig=rsOYXoYYlQdCXjDVZB5ufz_Vr0g&hl=en&sa=X&ved=0ahUKEwjAkdHClPbXAhVPMd8KHTd2BXkQ6AEIJzAA#v=onepage&q=Ignorance%3A%20inadequate%20security%20awareness%2C%20lack%20of%20security%20guidelines%2C%20lack%20of%20proper%20documentation%2C%20lack%20of%20knowledge&f=false

Reply:
Same comment as above